roseau/load_flow/models/buses.py

```diff
@@ -151,6 +162,22 @@ def res_voltages(self) -> Q_[ComplexArray]:
         """
         return self._res_voltages_getter(warning=True)
 
+    @property
+    def res_voltages_level(self) -> FloatArrayLike1D | None:
```

Comment:
The name should be `res_voltage_levels`. The type must be concrete as it is a return type `Q_[NDArray[np.float64]] | None`.